packages/bpk-component-datatable/src/bpkHeaderRenderer.js

```diff
@@ -37,16 +37,24 @@ const downIconClassName = getClassName(
   'bpk-data-table-column__sort-icon--down',
 );
 
-export const getSortIconDirection = (element: Element): ?SortDirectionType => {
+export const getSortIconDirection = (
+  eventTarget: EventTarget,
+): ?SortDirectionType => {
+  /* Flow does not work well with EventTarget type, but we need to use it in this case, because we need
```

Comment:
This is false AFAIK, Flow is correct and this code has a bug. 

[`EventTarget`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget) does not actually have a `parentElement` property which is why Flow is complaining.  In particular, all of `Window`, `Document`, and `Element` are examples of types that implement the `EventTarget` interface. However, `parentElement` is a property on `Node` which `Document` and `Element` are subclasses of but `Window` *is not*.

This code used `Element` precisely because it solves this problem as all `Element`s have a `parentElement` property

Reply:
You're right, done!